dimod/constrained.py

```diff
@@ -457,37 +488,43 @@ def add_discrete_from_comparison(self,
                                      comp: Comparison,
                                      label: Optional[Hashable] = None,
                                      copy: bool = True) -> Hashable:
-        """Add a one-hot constraint.
+        """Add a one-hot constraint from a comparison.
+
+        One-hot constraints can represent discrete variables (for example a
```

Comment:
I think maybe all of these functions should have have a ``See :meth:`add_constraint()` `` so we're not repeating ourselves too much.

Reply:
I considered doing that but the params slightly differ and I suspect there's a good chance that a user getting it work won't pop over to see the cautions on (1) supported vars and (2) LeapHybridCQMSampler behavior. 
If you're unhappy with repeating those two sentences though, I can have them just in ``add_discrete``.